OpenEdXMobile/src/main/java/org/edx/mobile/inapppurchases/BillingProcessor.kt

```diff
@@ -221,3 +222,5 @@ class BillingProcessor @Inject constructor(@ApplicationContext val context: Cont
         fun onPurchaseComplete(purchase: Purchase) {}
     }
 }
+
+fun SkuDetails.getPriceAmount(): Long = this.priceAmountMicros / BillingProcessor.MICROS_TO_UNIT
```

Comment:
The `priceAmount` will be a floating point number. So we need to update the dataType accordingly.